README.md

```diff
@@ -0,0 +1,41 @@
+# How to onboard new members
+
+1. Make sure new member has created accounts
+   - on [HCP Terraform](https://app.terraform.io)
+   - on [Hetzner Cloud](https://console.hetzner.cloud)
+   - on [Netlify](https://app.netlify.com)
+1. Invite new member
+   - to [our HCP Terraform organization](https://app.terraform.io/app/prismlauncher/settings/users/) into the `admins` team
+   - to [our Hetzner Cloud project](https://console.hetzner.cloud/projects/3927126/security/members) as `Admin`
+   - to [our Netlify team](https://app.netlify.com/teams/prismlauncher/members/new/access) as `Developer`
```

Comment:
I couldn't find out if Developers have access to DNS. @getchoo would you be willing to be my lab rat?